tests/components/test_rfxtrx.py

```diff
@@ -59,6 +59,16 @@ def test_valid_config2(self):
                 'dummy': True,
                 'debug': True}}))
 
+    def test_valid_config3(self):
+        """Test configuration."""
+        self.assertTrue(setup_component(self.hass, 'rfxtrx', {
+            'rfxtrx': {
+                'device': '/dev/serial/by-id/usb' +
+                          '-RFXCOM_RFXtrx433_A1Y0NJGR-if00-port0',
+                'dummy': True,
+                'debug': False,
+                'recv_modes': ['hideki', 'oregon', 'ac']}}))
+
```

Comment:
All the tests for Rfxtrx are disabled on line 13.
Since they are taking too long time.
Have you tested that they still pass locally?
(But it would be great if someone could optimize and re-enable the tests)

Reply:
No I did not. I will do that later as I do not have too much time right now.